modules/SpyUtils.js

```diff
@@ -4,6 +4,10 @@ import { isFunction } from './TestUtils'
 
 const noop = () => {}
 
+const engineSupportsConfigurableFunctionLength = (function () {
+  return Object.getOwnPropertyDescriptor(() => {}, 'length').configurable
```

Comment:
actually that check is not enough because of IE 8 having the method but only for DOM nodes